spec/helpers/breadcrumbs_helper_spec.rb

```diff
@@ -0,0 +1,11 @@
+require 'spec_helper'
+
+describe BreadcrumbsHelper do
+  describe '#page_breadcrumbs' do
+    it 'returns the right breadcrumb' do
+      expected_html = "<ul class=\"breadcrumb\"><li class=\"breadcrumb-item\">"\
+                    "<a href=\"http://test.host/account\">Dashboard</a></li></ul>"
```

Comment:
change `"<a href=\"/account\">Dashboard</a></li></ul>` to `"<a href=\"#{account_root_path}\">Dashboard</a></li></ul>`